.travis.yml

```diff
@@ -12,12 +12,6 @@ before_install:
 install:
   - yarn bootstrap
 
-cache:
```

Comment:
If you don't want to use the cache yarn: true then maybe add a comment saying "We explicitly disable this cache because X" so that no-one is tempted to add it back